google/cloud/spanner/value_test.cc

```diff
@@ -22,6 +22,15 @@
 
 namespace google {
 namespace cloud {
+
+void PrintTo(optional<bool> const& v, std::ostream* os) {
```

Comment:
Do you want to keep the edits in this file, or revert them?

Reply:
Reverting them seems like a better idea. Done.